routes/user.js

```diff
@@ -40,37 +40,38 @@ router.post("/add", function(req, res) {
     });
 });
 
-router.get("/verify", function(req, res) {
-  let email = gUnverifiedEmails.get(req.query.state);
+router.get("/verify", (req, res) => {
+  const email = gUnverifiedEmails.get(req.query.state);
   if (email) {
     gEmails.add(email);
     gUnverifiedEmails.delete(req.query.state);
-    res.json({ email, info: "Successfully added " + email});
+    res.json({ email, info: `Successfully added ${email}`});
     return;
   }
+  // TODO: Needs better error message.
   res.json({ info: "Who are you?" });
 });
 
-router.post("/remove", function(req, res) {
+router.post("/remove", (req, res) => {
   gEmails.delete(req.body.email);
   res.json({ email: req.body.email, info: "removed user" });
 });
 
-router.post("/reset", function(req, res) {
+router.post("/reset", (req, res) => {
   gEmails.clear();
   res.json({ info: "user list cleared" });
 });
 
 if (process.env.DEBUG_ALLOW_USER_LIST) {
```

Comment:
Do we need to whitelist this for eslint?

Reply:
Yeah, I can add `// eslint-disable-next-line no-process-env`, but wasn't sure if we were keeping those flags, or adding them to AppConstants, per https://github.com/mozilla/blurts-server/issues/36#issuecomment-364069848.

Reply:
Done!